examples/dataset-examples/good-maldiims/README.md

```diff
@@ -0,0 +1,3 @@
+```
```

Comment:
We haven't had end-to-end tests for all assay types, so don't feel obliged to add one here.

If this exercises an execution path that isn't sufficiently tested, great, but otherwise I would lean towards taking it out, if only because it adds a bit to the time it takes to run tests, without adding much. But this is only a slight preference.

Reply:
Sure, no problem - I only added it to make sure it wasn't failing